### PR TITLE
Microblog openhab added to dietpi-software

### PR DESCRIPTION
**Pull request by Anulo2**
closes https://github.com/MichaIng/DietPi-Website/issues/273

### CODE REVIEW

File: dietpi-software.html [hunk 1]
```diff
@@ -3724,6 +3756,13 @@ <h3>Synapse</h3>
 							<p>Matrix protocol server</p>
 						</a>
 					</div>
+					<div class="col-xl-2 col-lg-3 col-md-4 col-6 mix Social">
+						<a href="#softwareinfo" class="thumbnail" rel="#microblogpub">
+							<img src="images/dietpi-software/microblogpub-logo.png" alt="microblog.pub Logo" width="8" height="5" loadi	ng="lazy">
```
Comment (MichaIng) [inline dietpi-software.html:3761]:
```suggestion
							<img src="images/dietpi-software/microblogpub-logo.png" alt="microblog.pub Logo" width="8" height="5" loading="lazy">
```

File: dietpi-software.html [hunk 2]
```diff
@@ -2525,6 +2525,38 @@ <h3>Domoticz</h3>
 						</div>
 					</div>
 				</div>
+				<!-- Start details for portfolio project 206 openHAB-->
+				<div id="openhab" class="row single-project">
+					<div class="col-md-4"><a href="https://dietpi.com/docs/assets/images/dietpi-software-homeautomation-openhab.jpg" target="_blank" rel="noopener"><img src="images/dietpi-software/openhab-screenshot-small.jpg" alt="openHAB Screenshot" loading="lazy"></a></div>
```
Comment (MichaIng) [inline dietpi-software.html:2530]: The large versions of the screenshots are still missing in both cases, right? Put them into `images/dietpi-software/` dir and URL path as well.

Comment (Anulo2) [reply dietpi-software.html:2530]: no, the large version is the external link to the docs image. But maybe I'm not understanding completely what you mean

Comment (MichaIng) [reply dietpi-software.html:2530]: Ah sorry, I missed that they are pointing to the docs. All fine then.

**Review by MichaIng — APPROVED** [review]

**Pull request merged**